pkg/kubernetes/scripts/virtual-machines.js

```diff
@@ -71,8 +71,9 @@
         '$scope',
         'kubeLoader',
         'kubeSelect',
-        function($scope, loader, select) {
-            vmsReact.init($scope, loader, select);
+        'kubeMethods',
+        function($scope, loader, select, methods) {
+            vmsReact.init($scope, loader, select, methods);
```

Comment:
We need to analyze the need for keeping redux tree among user's jumping via Cockpit pages.
So far it's needed just for the error messages but maybe we find something else.
But it's for a follow-up.